### DISCUSSION TITLE
Automatic prevalence updates stopped on 2022-04-16

### DISCUSSION THREAD

**Discussion by Meowse** [microCOVID/microCOVID#1429] [Q&A] [answered]
The most recent automatic prevalence update was 2022-04-16 (see [automatic prevalence updates](https://github.com/microCOVID/microCOVID/pulls?q=is%3Apr+label%3Aauto-prevalence-update)).

1. Does anyone know what would be causing the automated prevalence updates to stop working? 
2. Is it possible to do a manual prevalence update while this issue gets resolved?

Also, I think we need a high-priority feature that shows a red flag in the UI whenever the prevalence data is more than a day or two out of date. With cases rising in many parts of the US, having information this far out of date is actively dangerous, because it will give people a false sense of safety. In Seattle, WA, for example, the prevalence numbers have gotten nearly 2x as bad since April 15th.

**Comment by Elipsis** [comment, marked as answer]
For now you can manually adjust by clicking "Override location-based data" and putting in the latest stats for "Reported cases in past week" and the other values.

And I agree that having a big warning when the data was out of date is a great suggestion!

**Comment by apiology** [comment]
Thanks, @Meowse! 

> Does anyone know what would be causing the automated prevalence updates to stop working?

Almost always this is caused by upstream sources changing formats or going through outages of their own.

> Also, I think we need a high-priority feature that shows a red flag in the UI whenever the prevalence data is more than a day or two out of date. 

That feature has since been added when things are over a week stale: https://github.com/microCOVID/microCOVID/pull/1437
